proto/tokenfactory/params.proto

```diff
@@ -4,4 +4,6 @@ package seiprotocol.seichain.tokenfactory;
 option go_package = "github.com/sei-protocol/sei-chain/x/tokenfactory/types";
 
 // Params defines the parameters for the tokenfactory module.
-message Params {}
+message Params {
+  int32 denom_allowlist_max_size = 1;
```

Comment:
Nit: consider using uint32 for easier validation

Reply:
done